tests/start_proxy/start_proxy_test.py

```diff
@@ -320,7 +320,7 @@ def test_gen_proxy_config(self):
               '--cors_preset', 'basic',
               '--cors_allow_origin', '*', '--cors_allow_origin_regex', '',
               '--cors_allow_methods', 'GET, POST, PUT, PATCH, DELETE, OPTIONS',
-              '--cors_allow_headers', 'X-Requested-With',
+              '--cors_allow_headers', 'DNT,User-Agent,X-User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range,Authorization',
```

Comment:
You removed the cors test flag setting. Add that back?

Reply:
I removed --cors_allow_headers flag passed to start_proxy.py,   in order to test its default values that passed to config_manager.

I can add another test to test the cors flags passed to start_proxy.py are correctly passed to config_manager.